Doc/whatsnew/3.7.rst

```diff
@@ -281,6 +281,14 @@ Function :func:`~uu.encode` now accepts an optional *backtick*
 keyword argument.  When it's true, zeros are represented by ``'`'``
 instead of spaces.  (Contributed by Xiang Zhang in :issue:`30103`.)
 
+xml.etree
+---------
+
+ElementPath predicates in the :meth:`find` methods can now compare text
```

Comment:
Make `ElementPath` a link:
```
:class:`~xml.etree.ElementPath`
```
Add a reference to `elementtree-xpath`:
```
:ref:`some text <elementtree-xpath>`
```

Reply:
I added a reference.